cmd/cli/devstack/devstack.go

```diff
@@ -250,3 +256,30 @@ func runDevstack(cmd *cobra.Command, ODs *devstack.DevStackOptions, IsNoop bool)
 	cmd.Println("\nShutting down devstack")
 	return nil
 }
+
+// getRepo will return a config repo that given a directory, will
+// create, init (if necessary) and open the repository before
+// returning it.
+func getRepo(repoPath string) (*repo.FsRepo, error) {
```

Comment:
Could we reuse this method instead?
https://github.com/bacalhau-project/bacalhau/blob/main/pkg/setup/setup.go#L69